Vic2ToHoI4/Source/HOI4World/Diplomacy/HoI4AIStrategy.h

```diff
@@ -0,0 +1,41 @@
+#ifndef HoI4AISTRATEGY_H_
+#define HoI4AISTRATEGY_H_
+
+
+
+#include "../../V2World/Ai/AIStrategy.h"
+
+
+
+namespace HoI4
+{
+
+class AIStrategy
+{
+  public:
+	explicit AIStrategy(std::string strategyType): type(std::move(strategyType)) {}
+
+	AIStrategy(const Vic2::AIStrategy& oldStrategy):
+		 type(std::move(oldStrategy.getType())), id(oldStrategy.getID()), value(oldStrategy.getValue())
+	{
+	}
+
+	void setID(const std::string HoI4Tag) { id = HoI4Tag; }
+	void setValue(const int strategyValue) { value = strategyValue; }
+	void updateStrategy();
```

Comment:
Incorporate the logic of this function into the relevant constructor then remove this function.